index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/index.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <p>Hello World</p>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div class="nav">
+      <ul>
+        <li class="listitems">
+          <a href="#">About</a>
+        </li>
+        <li class="listitems">
+          <a href="#">work</a>
+        </li>
+        <li class="listitems">
+          <a href="#">contact</a>
+        </li>
+      </ul>
+    </div>
+    <!-- start sectionone -->
+    <section class="sectionone pd-y" id="sectionone">
+      <div class="sectionone__wrapper">
+        <h1 class="" id="">Hey I am Mimic</h1>
+      <small><h3 class="center">a web developer</h3></small>
+      </div>
+    </section>
+    <!-- end sectionone -->
+    <!-- <h1>hello world and I am Mimic</h1>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis
+      laudantium excepturi dolorem rerum labore repellat voluptas debitis animi
+      culpa iusto!
+    </p> -->
```

Comment:
Again and Again, delete unused code as I told you up, so that your codebase with be clean, as leaving unused code in your codebase make it unclean and difficult to read through in other to find and fix a bug or understand the codebase

Reply:
Kindly be polite in your comments and suggestions instead of orders to sound more friendly as well as elaborate more on why you requested this change. You can refer to this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) on how you can communicate better when doing reviews.